editor/components/block-settings-menu/index.js

```diff
@@ -66,6 +66,7 @@ function BlockSettingsMenu( {
 						<BlockRemoveButton key="remove" uids={ uids } />,
 						<BlockDuplicateButton key="duplicate" uids={ uids } rootUID={ rootUID } />,
 						count === 1 && <ReusableBlockSettings key="reusable-block" uid={ uids[ 0 ] } onToggle={ onClose } />,
+						<div key="separator" className="editor-block-settings-menu__separator" />,
```

Comment:
We need to hide this line if no possible transformations exist maybe easier if we do it inside block-transformations.js.